js/main.ts

```diff
@@ -87,7 +87,7 @@ export default function denoMain() {
   log("cwd", cwd);
 
   // TODO handle shebang.
-  for (let i = 1; i < startResMsg.argvLength(); i++) {
+  for (let i = 0; i < startResMsg.argvLength(); i++) {
```

Comment:
This is intentional. I believe it serves the same purpose as your `let args = env::args().skip(1).collect();` line below